packages/angular/src/nos/nosStore.ts

```diff
@@ -0,0 +1,23 @@
+import {
```

Comment:
Is there a difference in functionality if we could include this in the store (angular-redux for example?), so we don't need to import our nOS store all the time?

Reply:
As you said in Discord "injection is react specific", which prevents me from using the `@nosplatform/api-functions/lib/react/nosStore.js`, because you are requiring `react` which is missing in the Angular package. This is the reason I created this file. In Angular I only need to export the functionality of `@nosplatform/api-functions/es6/bindings` and in `react`, I really don't know what's going on. `angular-redux` is kinda specific since there will be Vue and native JS starterkits as well, don't you think? I think we need to build them first and then unify the `nosSore` :-/ Well, I guess it's up to you to decide.

Reply:
Ok, I was just wondering if a Higher Order Component-like structure is ever used in Angular. But  if you're just importing it in your Angular components I'd prefer using the `index.js` from API functions instead of this file. Everything you use in *this file* is already defined there.

As your point on the unified store. I think it's a valid one, because I've been thinking of a unified compiler as well (see my remark on trying out Angular Parcel). It's something to look at in the future!

Reply:
I saw your PR for the new nOS functionalities being exported to more unified manner. I will use `index.js` once the PR is merged.